include/alpaka/acc/AccCpuOmp2Blocks.hpp

```diff
@@ -162,7 +166,21 @@ namespace alpaka
         {
             ALPAKA_FN_HOST static auto getAccName() -> std::string
```

Comment:
The PR headline mentioned a `constexpr` type demangler. Is it possible to make the function `constexpr` or `consteval`? In this case the performance of `std::format` and `std::stringstream` does not matter.

Reply:
I didn't find a way.

The performance doesn't matter much anyway, because the name is computed only once, the first time it's used.

Reply:
At the moment I play a little bit around with it but it is not easy that compiler really optimize.
But if the performance does not matter, we simply keep the simple implementation and optimize it, if become an bottleneck.

Reply:
Okay, I found a major problem, if we want to make it `constexpr`. C++23 is required for static variables in `constexpr` functions.

Reply:
Yes, it's the same issue I had to work around [here](https://github.com/fwyzard/alpaka/blob/fe19cdadf930d0adbd50448a99af08ae6fcba23c/include/alpaka/core/DemangleTypeNames.hpp#L75-L78):
```c++
    // Store the demangled type name as a null-terminated array of bytes.
    // Note: this could be a function-static constexpr variable in c++23.
    template<typename T>
    inline constexpr auto storage = demangleAsArray<T>();
```

Maybe we can use a `consteval` function and store it as a `static constexpr` data member.

Reply:
During the VC today we talked about the subject. It is not performance critical. Therefore we can keep it as it is.